frontend/src/components/objectTypes/ShowObjectTypesByHandler.jsx

```diff
@@ -0,0 +1,217 @@
+import React, { useState, useEffect, useReducer } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { useHistory } from 'react-router';
+import PropTypes from 'prop-types';
+
+import Typography from '@material-ui/core/Typography';
+import ListItem from '@material-ui/core/ListItem';
+import { makeStyles } from '@material-ui/core';
+import Link from '@material-ui/core/Link';
+
+import objectsAPI from '../../helpers/api/objects';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: 'calc(100% - 64px)',
+  },
+
+  inventory: {
+    fontSize: '0.875rem',
+    marginBottom: theme.spacing(2),
+  },
+
+  /* flex */
+  flexWrapper: {
+    padding: '0 16px',
+  },
+
+  flex: {
+    display: '-ms-flexbox',
+    // eslint-disable-next-line no-dupe-keys
+    display: 'flex',
+    position: 'relative',
+  },
+
+  /* Title row */
+  flexTitle: {
+    padding: '8px 0',
+    borderBottom: '2px solid #c0c0c0',
+  },
+
+  /* Data Row */
+  flexRow: {
+    padding: '8px 0',
+    borderBottom: '1px solid #c0c0c0',
+    '&:hover': {
+      boxShadow: '0 0 3px 0 rgba(0, 0, 0, 0.8)',
+    },
+  },
+
+  /* Column definiions */
+  flexCol1: {
+    flex: '1 1 0',
+    marginRight: '8px',
+    paddingLeft: '8px',
+    minWidth: '0',
+  },
+
+  flexCol2: {
+    flex: '2 1 0',
+    marginRight: '8px',
+    paddingLeft: '8px',
+    minWidth: '0',
+  },
+
+  flexCol3: {
+    flex: '3 1 0',
+    marginRight: '8px',
+    paddingLeft: '8px',
+    minWidth: '0',
+  },
+
+  flexCol4: {
+    flex: '4 1 0',
+    marginRight: '8px',
+    paddingLeft: '8px',
+    minWidth: '0',
+  },
+
+  flexColAuto: {
+    width: 'auto',
+    marginRight: '8px',
+    paddingLeft: '8px',
+    minWidth: '0',
+  },
+
+  /* Use to scale and hide columns in the title row */
+  flexColAutoHidden: {
+    width: 'auto',
+    marginRight: '8px',
+    paddingLeft: '8px',
+    minWidth: '0',
+    visibility: 'hidden',
+  },
+
+  show: {
+    display: 'block',
+  },
+
+  hide: {
+    display: 'none',
+  },
+
+  pointer: {
+    cursor: 'pointer',
+  },
+}));
+
+// change the state of toggleIds[objectType.id] and trigger React to update the screen
+// (see NOTE below)
+const handleToggles = (id, toggleId, setToggleIds, triggerUpdate) => {
+  const newIds = toggleId;
+  newIds[id] = !newIds[id];
+  setToggleIds(newIds);
+  triggerUpdate();
+};
+
+const ShowObjectTypesByHandler = ({ objectTypes, setIsLoading, setAlertProps }) => {
+  // NOTE: regarding toggleIds and triggerUpdate
+  // eslint-disable-next-line max-len
```

Comment:
I believe eslint complained when I had multi-line comments. (Also note I needed to add the "disable-next-line max-len" because the one of the comments is longer...)